node-scripts/spec/pipeline1/resources/default/.ocd3.yaml

```diff
@@ -0,0 +1,8 @@
+version: 1
+
+project:
+  name: bahmniapps
+build:
+  script: "file: ./scripts/build.sh"
+deploy:
+  script: ./scripts/deploy.sh
```

Comment:
I think another input that will be needed for the deploy stage is the location of the built artefacts. Maybe `artefacts-path`?